presto-main/src/main/java/com/facebook/presto/sql/gen/TryCodeGenerator.java

```diff
@@ -107,6 +110,7 @@ public static MethodDefinition defineTryMethod(
 
         method.comment("Try projection: %s", innerRowExpression.toString());
         method.getBody()
+                .append(wasNull.set(constantBoolean(false)))
```

Comment:
Is this a fix for another issue? It seems unrelated to the rest of the refactoring